de.metas.adempiere.adempiere/client/src/main/java/de/metas/process/ui/ProcessPanel.java

```diff
@@ -605,8 +605,15 @@ public void unlockUI(final ProcessInfo pi)
 			{
 				closeWindow = true;
 			}
+			
+			// #863
+			// Do not show result window for any process that doesn't allow rerun
+			if(!_allowProcessReRun)
```

Comment:
Hi @metas-rc , with this change, does the system still respect the `setShowProcessLogs()` setting we talked about?